doc/connectors/revinate.md

```diff
@@ -0,0 +1,52 @@
+# Revinate Connector
+
+Revinate connector cf. https://porter.revinate.com/documentation
+
+# Data Provider Configuration
+
+* `type`: `Revinate`
+* `name`: str, required
+* `authentication`: see below
+
+```coffee
+DATA_PROVIDERS: [
+    type: 'Revinate',
+    name: '<name>',
+    authentication: '<authentication>'
+]
+```
+
+# Revinate Authentication
+
+
+Please see the Revinate documentation to understand what form the headers must take before a request can be made to their API. This essential element of the connector must have the following form:
+
+* `api_key`: str, required
+* `api_secret`: str, required
+* `username`: str, required
+
+The `api_key`, `api_secret` and `user_name` must be obtained from Revinate before the concepteur can use the connector.
```

Comment:
```suggestion
The `api_key`, `api_secret` and `user_name` must be obtained from Revinate before using the connector.
```

"concepteur" is a toucan term, it's not understandable in the context of this open-source library